src/Microsoft.AspNet.Http.Interfaces/IRequestIdentifierFeature.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNet.Http.Interfaces
+{
+    /// <summary>
+    /// Feature to identify a request.
+    /// </summary>
+    public interface IRequestIdentifierFeature
+    {
+        /// <summary>
+        /// Identifier to trace a request.
+        /// </summary>
+        Guid TraceIdentifier { get; }
```

Comment:
I can't think of anything else we have in our entire stake that represents an Id that's a guid.